src/components/periodic-report/periodic-report.service.ts

```diff
@@ -84,25 +82,7 @@ export class PeriodicReportService {
       changes
     );
 
-    const { reportFile, ...simpleChanges } = changes;
-
-    const updated = await this.repo.updateProperties(current, simpleChanges);
-
-    if (reportFile) {
-      await this.files.updateDefinedFile(
-        current.reportFile,
-        'file',
-        input.reportFile,
-        session
-      );
-      const newVersion = await this.files.getFileVersion(
-        reportFile.uploadId,
-        session
-      );
-      await this.eventBus.publish(
-        new PeriodicReportUploadedEvent(updated, newVersion, session)
-      );
```

Comment:
I forgot about this. This is important. This is how we know to extract the pnp data. I think the best course of action is to add a `pnp: DefinedFile` to `ProgressReports` specifically. `reportFile` resolver could forward to this for progress reports.

The event we use internally can change. Maybe it becomes `PnpProgressUploadedEvent`.